service/utils.go

```diff
@@ -194,5 +196,6 @@ func showUpgradeVersion(version, upgradeVersion model.Version, rancherVersion st
 		}
 		return satisfiesRange
 	}
-	return true
+
+	return false
```

Comment:
The reason this was changed to false is because if there is no upgrade link, then that version shouldn't be shown in the upgrade section.